bin/clearIndex.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#:Title: clearIndex.sh
+#:Author: Alan Coleman
+
+#shut down logstash if running
+sh logstashScripts/stopLS.sh
+
+sleep 5
```

Comment:
This is only necessary if the scripts invoked beforehand are non-blocking / asynchronous, which I'm not sure they are. Even if they were, this is an extremely fragile approach. What if `stopLS.sh` takes longer than 5 seconds?

I would remove these unless they were proven necessary.

Reply:
tested it without the "sleep" lines and it doesn't appear to affect performance, will make this change